### PR TITLE
Fix readme doc errors of jca

### PR DESCRIPTION
**Pull request by backwind1233**
As title

### CODE REVIEW

**Comment by backwind1233** [comment]
Link with https://github.com/Azure/azure-sdk-for-java/issues/27329, finding doc errors why addressing the issue.

**Comment by backwind1233** [comment]
@chenrujun please help review this PR.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
    ```
    source script/setup.sh
    ```
-2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.   
+2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.
+   > Attention: The service principal must be configured with get and list permissions of certificate, key, secret.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:29]: For key and secret, only `get` permission is required.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md [hunk 2]
```diff
@@ -59,7 +59,8 @@ You can also create resources manually via Azure Portal. Please follow:
    - If you cannot use managed identity, you can register your application with AAD, see [register app with AAD][register_app_with_AAD]. The registration also creates a second application object that identifies your app.
 2. Create the key vault and certificates. Please refer to [create key vault and certificates][create_key_vault_and_certificates]
 3. Make the key vault accessible to your service principal. Please refer to [assign key vault access policy][assign_key_vault_access_policy]
-4. You need manually configure the application.yml, replace the placeholders with the resources you created in the Azure Portal.
+   > Attention: The service principal must be configured with get and list permissions of certificate, key, secret.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:62]: Same here.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md (hunk 2 above)
Comment (backwind1233) [reply keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:62]: 1. get and list are all read permission, I think it's ok to add list permission.
2. I want to be consistent with our shell
image

Comment (chenrujun) [reply keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:62]: I suggest to configure minimal permissions and update the shell script.

Comment (backwind1233) [reply keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md]: updated

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md [hunk 3]
```diff
@@ -25,7 +25,12 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
    ```
    source script/setup.sh
    ```
-2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.   
+2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:31]: Same here.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md [hunk 4]
```diff
@@ -25,7 +25,12 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
    ```
    source script/setup.sh
    ```
-2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.   
+2. Option 2 - If you created the resource via the Azure Portal, you need configure the application.yml manually, please replace the placeholders with the resources you created.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.  
+   > Secret Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md:32]: And here.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md [hunk 5]
```diff
@@ -59,7 +59,11 @@ You can also create resources manually via Azure Portal. Please follow:
    - If you cannot use managed identity, you can register your application with AAD, see [register app with AAD][register_app_with_AAD]. The registration also creates a second application object that identifies your app.
 2. Create the key vault and certificates. Please refer to [create key vault and certificates][create_key_vault_and_certificates]
 3. Make the key vault accessible to your service principal. Please refer to [assign key vault access policy][assign_key_vault_access_policy]
-4. You need manually configure the application.yml, replace the placeholders with the resources you created in the Azure Portal.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:64]: Here.

File: keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md [hunk 6]
```diff
@@ -59,7 +59,11 @@ You can also create resources manually via Azure Portal. Please follow:
    - If you cannot use managed identity, you can register your application with AAD, see [register app with AAD][register_app_with_AAD]. The registration also creates a second application object that identifies your app.
 2. Create the key vault and certificates. Please refer to [create key vault and certificates][create_key_vault_and_certificates]
 3. Make the key vault accessible to your service principal. Please refer to [assign key vault access policy][assign_key_vault_access_policy]
-4. You need manually configure the application.yml, replace the placeholders with the resources you created in the Azure Portal.
+   > Attention: The service principal must be configured with permissions:   
+   > Certificate Permissions: configure with **get and list** permissions.  
+   > Key Permissions: configure with get permission.  
+   > Secret Permissions: configure with get permission.
```
Comment (chenrujun) [inline keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md:65]: Here.

**Review by chenrujun — APPROVED** [review]

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**